src/main/java/github/scarsz/discordsrv/objects/managers/link/FileAccountLinkManager.java

```diff
@@ -145,8 +145,11 @@ public String process(String linkCode, String discordId) {
             }
         }
 
-        // strip the code to get rid of non-numeric characters
-        linkCode = linkCode.replaceAll("[^0-9]", "");
+        // strip the code to get rid of non-numeric and non-alphabetical characters
+        linkCode = linkCode.replaceAll("[^A-Z0-9]", "");
+        // remove case sensitivity
+        linkCode = linkCode.toUpperCase();
```

Comment:
Please move the `toUpperCase` call above the regex, otherwise lowercase characters will not be picked up